### PR TITLE
Adding Ocetrac

### PR DESCRIPTION
**Pull request by hscannell**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hscannell** [comment]
@conda-forge-admin this is ready for review. 
I can confirm that I am the maintainer.

File: recipes/ocetrac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - ocetrac
+    - ocetrac.track
+
+about:
+  home: https://github.com/ocetrac/ocetrac
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (jbusecke) [inline recipes/ocetrac/meta.yaml:44]: I think this should be updated?

File: recipes/ocetrac/.ipynb_checkpoints/meta-checkpoint.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (jbusecke) [inline recipes/ocetrac/.ipynb_checkpoints/meta-checkpoint.yaml:1]: I believe this checkpoint file should not be in the PR.

File: recipes/ocetrac/meta.yaml (hunk 1 above)
Comment (hscannell) [reply recipes/ocetrac/meta.yaml]: Good catch @jbusecke !

File: recipes/ocetrac/.ipynb_checkpoints/meta-checkpoint.yaml (hunk 2 above)
Comment (hscannell) [reply recipes/ocetrac/.ipynb_checkpoints/meta-checkpoint.yaml]: Done, thank you.

**Pull request closed without merging**